addons/info/src/components/PropTable.js

```diff
@@ -1,5 +1,6 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import 'core-js/fn/array/includes';
```

Comment:
You need to add `core-js` as dependency to `addons/info/package.json`

Reply:
Added